Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_ObjectFollow.cs

```diff
@@ -13,8 +13,15 @@ public abstract class VRTK_ObjectFollow : MonoBehaviour
         [Tooltip("The game object to change the property values of. If left empty the game object this script is attached to will be changed.")]
         public GameObject gameObjectToChange;
 
+        [Tooltip("Whether to follow the position. CAUTION: This value overwrite gameObjectToFollow position! Set to Vector3.zero for use gameObject method")]
+        public Vector3 positionToFollow = Vector3.zero;
```

Comment:
Since the `positionToFollow`  can't be used with `gameObjectToChange` it should probably be changed somehow... Maybe a wrapper component or something, not sure currently. 🤔